libadikted/Makefile

```diff
@@ -43,9 +43,9 @@ else
   DEBUGME = no
 endif
 # Names of utility commands
-WARNFLAGS          = -Wall -Wextra -Wno-pedantic -Wno-conversion -Wno-traditional-conversion -Wno-sign-compare
```

Comment:
I welcome improved warnings, however I would have prefered it to be part of a seperate PR to avoid noise.

Reply:
Ok I will move warnings to another PR

Reply:
Even two of them #13 and #14